src/main/java/org/apache/commons/imaging/formats/jpeg/iptc/IptcRecord.java

```diff
@@ -38,18 +38,22 @@ public IptcRecord(final IptcType iptcType, final byte[] bytes, final String valu
         this.value = value;
     }
 
-    public IptcRecord(final IptcType iptcType, final String value) {
+    public IptcRecord(final IptcType iptcType, final String value, final String charsetName) {
         this.iptcType = iptcType;
         byte[] tempBytes;
         try {
-            tempBytes = value.getBytes("ISO-8859-1");
+            tempBytes = value.getBytes(charsetName);
```

Comment:
Overall I think the code looks good to be merged. I would need to find some spec/docs for reference to clear up a few more questions about the issue.

But the main issue right now are the conflicts. The `tempBytes` was removed from `IptcRecord`, and now we have only the `String value`. We need to look at the git history and past issues, and try to understand why it was changed this way.

Then we can resolve the conflicts and leave it ready to be merged :+1: